### PR TITLE
Add  rich text (wysiwyg) editor

### LINKED ISSUE #125: Add Rich Text to Toolbar
I want be able to be able to drag a rich text WYSIWYG block from the controls onto the canvas. 

Use this icon: pencil-ruler
label: Rich Text

### PR DESCRIPTION
**Pull request by chipit24**
Fixes #125.

**Important changes:**
* `eslint-plugin-jest` has been added; run `npm ci` or `npm i` before testing.
* New test added; all tests should pass when running `npm test`.

rich-text

### CODE REVIEW

File: src/components/vue-form-builder.vue [hunk 1]
```diff
@@ -165,6 +165,8 @@
 
   import BootstrapVue from "bootstrap-vue";
 
+  import '@processmaker/vue-form-elements/dist/vue-form-elements.css';
```
Comment (chipit24) [inline src/components/vue-form-builder.vue]: If having a separate CSS import is preferred, styles could be inlined. See https://cli.vuejs.org/guide/build-targets.html#library:
> * `dist/myLib.css`: Extracted CSS file (can be forced into inlined by setting `css: { extract: false }` in `vue.config.js`)

**Comment by velkymx** [comment]
AH! OK the editor should only show in the edit screen. The preview screen should only show the final resulting HTML

**Comment by chipit24** [comment]
@velkymx I've updated component to work as expected:
rich_text_editor

**Note:** It requires the changes from x to be merged and published; blocked until then.

**Comment by velkymx** [comment]
@chipit24 @milaendo can we add default text to the control. It's confusing if there isn't anything there.

**Comment by milaendo** [comment]
@chipit24 @velkymx After digging into this I have run into an interesting bug. Ill do my best to explain:

in the form-builder-controls.js There is "content" being set in the config for the html editor, which should show on the canvas when the control is dragged on as the default text. However, in the vue-form-elements the value is being set equal to the content. in the config if I add "value: click here to edit", it will show as the default text but I am unable to edit it in both the inspector as well as the canvas.

Im happy to jump on a call if this doesnt make sense.

**Comment by chipit24** [comment]
I've updated the PR.

@milaendo It's possible that this PR wasn't referencing the latest `@processmaker/vue-form-elements` package when you tested it. Please pull down the latest changes, make sure to run `npm i` or `npm ci`, and test again. If it's still failing we can hop on a call or you can post a video of the problem.

**Comment by milaendo** [comment]
@chipit24 That seems to have been the problem! all is fixed :) my only feedback now is that the control icon is a bit larger than all of the others 
Screen Shot 2019-04-01 at 11 06 06 AM

**Comment by chipit24** [comment]
Updated styles:

Screen Shot 2019-04-01 at 3 18 05 PM

**Comment by velkymx** [comment]
@chipit24 please resolve these conflicts to merge.

**Comment by velkymx** [comment]
@milaendo @chipit24 Move the control to the top of the menu and let's merge it.

**Comment by chipit24** [comment]
@velkymx Done.

**Pull request merged**